metadata-ingestion/src/datahub/utilities/sqlalchemy_query_combiner.py

```diff
@@ -185,9 +195,7 @@ def _handle_execute(
 
         # Figure out how many columns this query returns.
         # This also implicitly ensures that the typing is generally correct.
-        if not hasattr(query, "columns"):
```

Comment:
We are experiencing some weird errors during profiling that were not present before this PR. Basically, our query returns a string object and hence fails the assertion. Before, _handle_execute() catched this situation by returning False, None in this case.